### DISCUSSION TITLE
Invite only SignUp [Aug 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3124] [Discord Threads]
> 🙋‍♂️ **vieiraemanuell**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@568238229711749128> You will be required to configure environment variable NC_INVITE_ONLY_SIGNUP   
####    
#### https://docs.nocodb.com/getting-started/installation#production-setup  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1004556519800578149)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
